src/constants/index.ts

```diff
@@ -308,6 +308,7 @@ export const APP_PATHS = {
   SWAP_LEGACY: '/swap-legacy',
   ABOUT: '/about',
   SWAP: '/swap',
+  SWAP_V3: '/swap-v3',
```

Comment:
/swap-new is better?

Reply:
This is fine because it's not visible to users. It will be '/swap' once we're done with this temporary page